### PR TITLE
Added Ubuntu 22.04 install instructions

### PR DESCRIPTION
**Pull request by bjauny**
Just a small update to the README. Ubuntu instructions differ a little due to different default packages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ To install vumz, you need to have `ncurses` and `pipewire` installed on your sys
 sudo dnf install pipewire pipewire-devel ncurses ncurses-devel
 ```
 
+### Ubuntu (22.04+)
```
Comment (IonelPopJara) [inline README.md:22]: Looks good to me. Thank you for adding it. I hope the program works fine!

**Review by IonelPopJara — APPROVED** [review]
Thanks for adding the Ubuntu instructions. I hope the program works fine! Feel free to press the `merge` button whenever you can.

**Comment by bjauny** [comment]
> Thanks for adding the Ubuntu instructions. I hope the program works fine! Feel free to press the `merge` button whenever you can.

Actually I can't, I don't have the write access to your repo. :)

**Comment by IonelPopJara** [comment]
> > Thanks for adding the Ubuntu instructions. I hope the program works fine! Feel free to press the `merge` button whenever you can.
> 
> Actually I can't, I don't have the write access to your repo. :)

my bad, I'll merge it then. Thank you!

**Pull request merged**